analyzer/src/funTest/assets/projects/synthetic/bundler-expected-output.yml

```diff
@@ -2,6 +2,7 @@
 allowDynamicVersions: false
 project:
   id: "Bundler::lockfile:"
```

Comment:
Do we want to hard-code the path here or use `<REPLACE_PATH>` and `patchExpectedResult()` as we do with the VCS info?

Reply:
We currently have hard-coded the (relative) path in most files, and only use the patching mechanism if required, i.e. if the same expected result is shared across multiple tests. Thus I'd prefer to keep it as-is until we actually need to change it.

Reply:
I see. I thought it was to avoid fixing it everywhere if we change the directory structure.